frontend_tests/casper_tests/03-narrow.js

```diff
@@ -367,6 +404,48 @@ casper.waitForSelector('.input-append.notdisplayed', function () {
 
 un_narrow();
 
+// User search at the right sidebar
+casper.then(function () {
+    casper.test.info('Search users using right sidebar');
+    casper.test.assertExists('#user_presences li [data-name="Cordelia Lear"]', 'User Cordelia Lear exists');
+    casper.test.assertExists('#user_presences li [data-name="King Hamlet"]', 'User King Hamlet exists');
+    casper.test.assertExists('#user_presences li [data-name="aaron"]', 'User aaron exists');
+});
+
+// Enter the search box and test selected suggestion navigation
+// Click on search icon
+casper.then(function () {
+    casper.evaluate(function () {
+        $('#user_filter_icon').expectOne()
+            .focus()
+            .trigger($.Event('click'));
+    });
+});
+
+casper.waitForSelector('#user_presences .highlighted_user', function () {
+    casper.test.info('Suggestion highlighting - initial situation');
+    casper.test.assertExist('#user_presences li.highlighted_user [data-name="Cordelia Lear"]', 'User Cordelia Lear is selected');
```

Comment:
I'd line-wrap these; we're planning to start enforcing maximum line lengths for our JS codebase, and it'll save a bit of work then.